zdm/misc_functions.py

```diff
@@ -1761,11 +1764,11 @@ def get_zdm_grid(new=True,plot=False,method='analytic',
 		plt.tight_layout()
 		plt.savefig('p_dm_slices.pdf')
 		plt.close()
-		
-		
-	return zDMgrid, zvals,dmvals
 
-def plot_zdm_basic_paper(zDMgrid,zvals,dmvals,zmax=1,DMmax=1000,norm=0,log=True,name='temp.pdf',ylabel=None,label='$\\log_{10}p(DM_{\\rm EG},z)$',project=False,conts=False,FRBZ=None,FRBDM=None):
+	#return zDMgrid, zvals,dmvals    
+    return zDMgrid, zvals,dmvals, H0
+
+def plot_zdm_basic_paper(zDMgrid,zvals,dmvals,zmax=1,DMmax=1000,norm=0,log=True,name='temp.pdf',ylabel=None,label='$\\log_{10}p(DM_{\\rm EG},z)$',project=False,conts=False,FRBZ=None,FRBDM=None,title='Plot',H0=cos.DEF_H0):
```

Comment:
Can you take away the H0 part in the function call, since I think you only use it to define the title, and since "title" gets passed anyway, somebody could just call the function with title=old_title+"_"+str(H0)

Reply:
I could do that, but then the `H0` information gets lost - if this function called the `grid` then I could easily extract the `H0 `value from it. However since it calls` zDMgrid, zvals` and `dmvals` there is no way to figure out what the value of` H0` is!